src/descriptor/create_descriptor.rs

```diff
@@ -292,6 +292,17 @@ mod tests {
         (pks, sigs, msg, secp_verify)
     }
 
+    fn des_str(s: &str) -> Descriptor<PublicKey, hash160::Hash>
+    {
+        Descriptor::<PublicKey, hash160::Hash>::from_str(s).unwrap()
+    }
+
+    fn ms_str(s: &str) -> Miniscript<PublicKey, hash160::Hash>
+    {
+        Miniscript::<PublicKey, hash160::Hash>::from_str(s).unwrap()
+    }
```

Comment:
Can you make these macros so that users don't have to do `&format!` themselves?

Reply:
@apoelstra How about another file, util.rs for the utility functions and separating them from lib.rs.
Maybe the macros go there

Reply:
Sure, or we could just throw them in `lib.rs` with a `#[cfg(test)]` on them. Whatever you think is cleaner